.gitignore

```diff
@@ -0,0 +1 @@
+.DS_Store
```

Comment:
[Fix] gitignore trebuie sa contina si .idea si fisierul .iml, cat si folderul /out, ca sa fii sigur ca nu ii faci commit

Reply:
Am facut repo-ul in directorul unde se afla codul acesta, inteleg ce spuneti si acum inteleg cum trebuia sa fac, dar nu stiu cum sa repar.